integrations-examples/twilio/app.js

```
@@ -31,83 +31,83 @@ const wss = new WebSocket.Server({ server });
+      case 'connected':
         console.log(`A new call has started`);
         break;
-      case "start":
-        console.log("Starting media stream...");
+      case 'start':
+        console.log('Starting media stream...');
         break;
 
-      case "media":
+      case 'media':
         const twilioData = msg.media.payload;
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)